tests/integration/standard/test_metadata.py

```diff
@@ -1277,13 +1277,13 @@ def test_already_exists_exceptions(self):
         cluster.shutdown()
 
     @local
-    @pytest.mark.xfail(reason='AssertionError: \'RAC1\' != \'r1\' - probably a bug in driver or in Scylla')
```

Comment:
I would prefere to create an issue and investigate what is going on here, why on scylla driver get's different rack name ?

Reply:
it's ccm code

scylla default to `GossipingPropertyFileSnitch` and the logic there uses RAC1
cassandra default to `PropertyFileSnitch` and the logic default to r1

we can align it on ccm, if you insist